packages/react-auth/src/AuthNotifications.tsx

```diff
@@ -0,0 +1,60 @@
+import * as React from 'react';
+import { Button, Flex } from '@ttoss/ui';
+import {
+  type NotifyParams,
+  useNotifications,
+} from '@ttoss/react-notifications';
+
+export const AuthNotifications = () => {
+  const { setNotifications, notifications } = useNotifications();
+
+  if (!notifications) {
+    return null;
+  }
+
+  const ButtonMemoized = React.memo(({ message, type }: NotifyParams) => {
+    return (
+      <Button
+        sx={{
+          backgroundColor: type === 'error' ? 'danger' : 'positive',
+        }}
+        onClick={() => {
+          return setNotifications(undefined);
+        }}
+        rightIcon="close"
+        leftIcon={type === 'error' ? 'warning' : undefined}
+      >
+        {message}
+      </Button>
+    );
+  });
```

Comment:
This Button is gonna be switched by Toast component when it's ready

Task: https://app.clickup.com/t/30901592/TASK-779